modules/investigating-why-windows-machine-compute-node.adoc

```diff
@@ -18,5 +18,5 @@ There are various reasons why a Windows Machine does not become a compute node.
 +
 [source,terminal]
 ----
-$ oc logs -f $(oc get pods -o jsonpath={.items[0].metadata.name} -n openshift-windows-machine-config-operator) -n openshift-windows-machine-config-operator
+$ oc logs -f deployment/windows-machine-config-operator) -n openshift-windows-machine-config-operator
```

Comment:
@EricPonvelle I think you need to remove the closing parenthesis after _operator_, based on the linked MD file.

`$ oc logs -f deployment/windows-machine-config-operator -n openshift-windows-machine-config-operator
`